### DISCUSSION TITLE
Struggling to install and setup Eufy Security

### DISCUSSION THREAD

**Discussion by anf8701** [fuatakgun/eufy_security#689] [General]
HI All, really struggling to setup Eufy Security. For some reason it doesn't let me start RTSP Simple Server Add on. When i go to add the integration eufy security it asks for a host IP and port. Nothing works no matter what i put in. I suspect it is because i can't start the RTSP simple server (not sure). Any help would be appreciated.

**Comment by fuatakgun** [comment]
Please use home assistant community forum to ask general help, it would have faster turnaround
